src/components/Header/MobileMenu.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react"
+import { PropTypes } from "prop-types"
+import { NavLink } from 'react-router-dom'
+
+import LogoHead from "./LogoHead"
+import PointsBal from "./PointsBal"
+import Navigation from "./Navigation"
+import {
+    ContainerHeader,
+    ExitIconWrapper,
+    LeftBlock,
+    MenuHeaderBlock,
+    MenuInitiaLetterCircle,
+    MenuInitiaLetter,
+    MenuName,
+    MobileHamburgerContainer,
+    MobileHamburgerLine,
+    MobileHamburgerbackground,
+    MobileHamburgerLoginBox
+} from "./Header.styled"
+import ExitIconMobile from "../../assets/image/Header-exit-icon-mobile.svg"
+
+const MobileMenu = (props) => {
+    const logoutAction = () => { props.logoutProps() }
```

Comment:
same here

Reply:
I removed the props passing from this file and the logout click event from the parent. Now this function and event are in this file))
Picture1